quartz/plugins/emitters/componentResources.ts

```diff
@@ -201,6 +201,36 @@ function addGlobalPageResources(ctx: BuildCtx, componentResources: ComponentReso
       })(window, document, "clarity", "script", "${cfg.analytics.projectId}");\`
       document.head.appendChild(clarityScript)
     `)
+  } else if (cfg.analytics?.provider === "matomo") {
+    const siteId = cfg.analytics.siteId
+    const matomoHost = cfg.analytics.host
+    componentResources.afterDOMLoaded.push(`
+      const matomoScript = document.createElement("script");
+      matomoScript.innerHTML = \`
+      var _paq = window._paq = window._paq || [];
+      var currentUrl = location.href;
+      // https://developer.matomo.org/guides/spa-tracking
+
+      // Track SPA navigation
+      document.addEventListener("nav", () => {
+        _paq.push(['setCustomUrl', location.pathname]);
+        _paq.push(['setDocumentTitle', document.title]);
+        _paq.push(['trackPageView']);
+      });
+
+      _paq.push(['trackPageView']);
+      _paq.push(['enableLinkTracking']);
+      (function() {
+        var u="//${matomoHost}/";
+        _paq.push(['setTrackerUrl', u+'matomo.php']);
+        _paq.push(['setSiteId', ${siteId}]);
+        var d=document, g=d.createElement('script'), s=d.getElementsByTagName
```

Comment:
Prefer `const` over `var`.

Reply:
@saberzero1 Thanks, change to `const` (or `let _paq`)